trino-connector/trino-connector/src/main/java/org/apache/gravitino/trino/connector/util/GeneralDataTypeTransformer.java

```diff
@@ -103,7 +104,11 @@ public Type getTrinoType(org.apache.gravitino.rel.types.Type type) {
       case TIME:
         return TimeType.TIME_MILLIS;
       case TIMESTAMP:
-        return TimestampType.TIMESTAMP_MILLIS;
+        if (((Types.TimestampType) type).hasTimeZone()) {
+          return TimestampWithTimeZoneType.TIMESTAMP_TZ_MILLIS;
```

Comment:
why use MILLIS as default?

Reply:
The Trino default time  is MILLIS

Reply:
could you provide related document or code?

Reply:
https://trino.io/docs/current/language/types.html#date

Reply:
got it